index.html

```diff
@@ -127,7 +128,259 @@ <h2>Multi-Post Stories</h2>
+                        <li class="projectBootstrap">
+                            <ul>
+                                <li>bootstrap</li>
+                            </ul>
+                        </li>
+                        <li class="projectRuby">
+                            <ul>
+                                <li>ruby</li>
+                            </ul>
+                        </li>
+                    </ul>
+                </div>
+                <div class="projectButton">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </section>
+
+        <section class="whoIsNemwel">
+            <div class="Nemweldesc">
+                <div class="nemweldescleft">
+                    <div class="nemwelHeader">
+                        <h2>About me</h2>
+                    </div>
+                    <div class="nemwelbody">
+                        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                        <div class="nemwelbtn">
+                            <button type="button">Get My Resume</button>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="nemweldescright">
+                    <img src="images/aboutmebgimage.png" alt="aboutme background image">
+                </div>
+                <hr>
+            </div>
+
+            <div class="nemwelskills">
+                <div class="skill">
+                    <div class="nemwelskillheader">
+                        <p class="lang"></p>
+                        <h2 class="language">Language</h2>
+                    </div>
+                    <div class="skilllist">
+                        <ul>
+                            <li>Javascript</li>
+                            <li>Ruby</li>
+                            <li>Html</li>
+                            <li>Css</li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="skill">
+                    <div class="nemwelskillheader">
+                        <p class="frame"></p>
+                        <h2 class="frameworks">Frameworks</h2>
+                    </div>
+                    <div class="skilllist">
+                        <ul>
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSPec</li>
+                            <li>CapyBara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="skill">
+                    <div class="nemwelskillheader">
+                        <p class="sk"></p>
+                        <h2 class="skills">Skills</h2>
```

Comment:
- [ ] Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. This will make your code more semantic.

This is important because structural, semantic HTML is the key starting point toward good accessibility practices.

Reply:
Thank you @mike2611 for this review.
- Ihave made the necessary changes